corehq/apps/userreports/transforms/specs.py

```diff
@@ -75,3 +75,16 @@ def transform_function(value):
                 return value
 
         return transform_function
+
+
+class TranslationTransform(Transform):
+    type = TypeProperty('translation')
+    translations = DictProperty()
```

Comment:
might be worth making this a proper `JsonObject` subclass that adds structure to the translations.